src/System.IO/tests/TextWriter/TextWriterTests.cs

```diff
@@ -0,0 +1,598 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class TextWriterTests
+    {
+        protected static string strfmtOneObject = "Object is {0}";
+        protected static string strfmtTwoObjects = $"Object are '{0}', {secondObject}";
+        protected static string strfmtThreeObjects = $"Objects are {0}, {secondObject}, {thirdObject}";
+        protected static string strfmtMultipleObjects = "Multiple Objects are: {0}, {1}, {2}";
+
+        protected static object firstObject = (object)1;
+        protected static object secondObject = (object)"[second object]";
+        protected static object thirdObject = (object)"<third object>";
+        protected static object[] multipleObjects = new object[] { firstObject, secondObject, thirdObject };
+
+
+        protected char[] GetCharArray()
+        {
+            return new char[]{
+                char.MinValue
+                ,char.MaxValue
```

Comment:
what's with the commas? we prefer them at the end of the line.

Reply:
hmmm. I copied this block from [here](https://github.com/dotnet/corefx/blob/59915687bb261efeae70565d0ca8990ac45ceb9c/src/System.IO/tests/StreamReader/StreamReaderTests.cs#L41-L69) and continued to follow the same style as per the existing code...

Do you want me to change in both the places...?

Reply:
done just in this place :) ....

Reply:
if you could fix both that would be great.

Reply:
done in 4 places... through out the solution...